src/Traits/IsAuthorizedTrait.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Copyright 2010 - 2019, Cake Development Corporation (https://www.cakedc.com)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright Copyright 2010 - 2018, Cake Development Corporation (https://www.cakedc.com)
+ * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
+ */
+
+namespace CakeDC\Auth\Traits;
+
+use CakeDC\Auth\Rbac\Rbac;
+use Cake\Http\ServerRequest;
+use Cake\Routing\Exception\MissingRouteException;
+use Cake\Routing\Router;
+use Zend\Diactoros\Uri;
+
+trait IsAuthorizedTrait
+{
+    /**
+     * Returns true if the target url is authorized for the logged in user
+     *
+     * @param string|array|null $url url that the user is making request.
+     *
+     * @return bool
+     */
+    public function isAuthorized($url = null)
+    {
+        if (empty($url)) {
+            return false;
+        }
+
+        if (is_array($url)) {
+            return $this->checkRbacPermissions(Router::normalize(Router::reverse($url)));
```

Comment:
@rochamarcelo In my opinion, it would be best to retrieve the current application mapped `CollectionPolicy` to take into account all the policies. For now, when `SuperuserPolicy` is applied (by default), it is ignored by `AuthLinkHelper` because it checks only the RBAC permissions resulting in links being hidden if not explicitly allowed (forced to be shown) :wink: